pcsx2/CDVD/CdRom.cpp

```diff
@@ -844,6 +841,26 @@ void cdrWrite1(u8 rt)
 			cdr.Mode = cdr.Param[0];
 			cdr.Ctrl |= 0x80;
 			cdr.Stat = NoIntr;
+			if (cdr.Mode && MODE_CDDA)
+			{
+				StopCdda();
+				cdvd.Type = CDVD_TYPE_CDDA;
+				if (cdr.Reading == 1)
+				{
+					StopReading();
+				}
+				if (cdr.SetlocPending)
+				{
+					memcpy(cdr.SetSectorSeek, cdr.SetSector, 4);
```

Comment:
Why do you need a memcpy? Can't you simply make a static allocation or a bitmask if you only want 4 bytes?